### PR TITLE
Feature add docker

### PR DESCRIPTION
**Pull request by pnieto**

### CODE REVIEW

**Comment by rubenrua** [comment]
I want review the pull request.

File: doc/docker/Makefile [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (rubenrua) [inline doc/docker/Makefile]: Empty???

File: doc/docker/nginx/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+FROM alpine:latest
+
+RUN apk add --no-cache openssl
+
+# Use this self-generated certificate only in dev, IT IS NOT SECURE!
+RUN openssl genrsa -des3 -passout pass:NotSecure -out cert.pass.key 2048
+RUN openssl rsa -passin pass:NotSecure -in cert.pass.key -out cert.key
+RUN rm cert.pass.key
+RUN openssl req -new -passout pass:NotSecure -key cert.key -out cert.csr \
+    -subj '/C=SS/ST=SS/L=Gotham City/O=API Platform Dev/CN=localhost'
```
Comment (rubenrua) [inline doc/docker/nginx/Dockerfile]: API platform in Gotham city???

File: doc/docker/nginx/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+FROM nginx:${NGINX_VERSION}-alpine
+
+RUN mkdir -p /etc/nginx/ssl/
+
+COPY --from=0 cert.key cert.crt /etc/nginx/ssl/
+COPY conf/default.conf /etc/nginx/conf.d/default.conf
+
+WORKDIR /srv/pumukit
+
+COPY --from=teltek/pumukit2:latest /srv/pumukit/web web/
```
Comment (rubenrua) [inline doc/docker/nginx/Dockerfile]: What about the dynamic assets???

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+version: '3.4'
+
+services:
+  php:
+    image: teltek/pumukit2
```
Comment (rubenrua) [inline docker-compose.yml]: Why not `teltek/pumukit` without 2?

File: doc/docker/nginx/Dockerfile (hunk 3 above)
Comment (pnieto) [reply doc/docker/nginx/Dockerfile]: shared dynamic assets with the volumes defined in the docker-compose file

**Comment by pnieto** [comment]
WIP: dev environment

**Comment by rubenrua** [comment]
`mis dieses`

Green light.

**Comment by Alfro** [comment]
Thanks!

**Pull request merged**